spec/controllers/announcement_controller_spec.cr

```diff
@@ -78,6 +78,18 @@ describe AnnouncementController do
       expect(response.status_code).to eq 200
       expect(response.body.includes? "Make an Announcement").to be_true
     end
+
+    context "rate limit reached" do
+      before { login_as user }
+
+      it "shows rate limit message" do
+        ENV["MAX_ANNS_PER_HOUR"] = "0"
+        get "/announcements/new"
+        expect(response.status_code).to eq 200
+        expect(response.body.includes? "You can create up to").to be_true
+        ENV["MAX_ANNS_PER_HOUR"] = nil
```

Comment:
I think `ENV["MAX_ANNS_PER_HOUR"] = "0"` should be moved to before and `ENV["MAX_ANNS_PER_HOUR"] = nil` to after block. Because if test fails, env variable will stay modified.

Reply:
You're right, I was thinking in use an ```ensure``` block to avoid this, but ```after``` hook is better!

Reply:
With https://github.com/crystal-lang/crystal/pull/5114 it will be possible to do:

```crystal
      it "shows rate limit message" do
        ENV["MAX_ANNS_PER_HOUR"] = "0"
        get "/announcements/new"
        expect(response.status_code).to eq 200
        expect(response.body.includes? "You can create up to").to be_true
      ensure
        ENV["MAX_ANNS_PER_HOUR"] = nil
      end
```

which is quite nice :)

Reply:
This syntax is awesome!